app/views/results/exceptionsResults.scala.html

```diff
@@ -0,0 +1,97 @@
+@*
+* Copyright 2016 LinkedIn Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not
+* use this file except in compliance with the License. You may obtain a copy of
+* the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations under
+* the License.
+*@
+
+@(result: com.linkedin.drelephant.exceptions.HadoopException)
+
+@import com.linkedin.drelephant.exceptions.HadoopException
+
+
+
+@tags.panel() { Exception Results} {
+  <head>
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+  </head>
+  <body>
+    <div>
+
+      <a data-toggle="collapse" href="#level1">Exception Analysis Result</a>
+      <div class="panel-collapse collapse" id="level1">
+        <ul class="list-group">
+
+        @if(result != null && !result.getChildExceptions().isEmpty()) {
+          @* <font color = "red"> <li class="list-group-item" >
+            The execution is unsuccessful with following azkaban job failure: </li> </font> *@
+          @for(azkabanJobException <- result.getChildExceptions()) {
+            <font color = "red"> <li class="list-group-item"> <a data-toggle="collapse" href="#@azkabanJobException.getId()" > Failed Azkaban Job: @azkabanJobException.getId()</a></li> </font>
+            <div class="panel-collapse collapse" id=@azkabanJobException.getId()>
+              <ul class = "list-group">
+                  @if(azkabanJobException.getType() == HadoopException.HadoopExceptionType.AZKABAN) {
+                    <li class="list-group-item"> <a data-toggle="collapse" href="#azkaban@azkabanJobException.getId()">
+                        &nbsp;&nbsp;&nbsp;Azkaban Level Failure</a></li>
+                    <div class="panel-collapse-collapse" id=azkaban@azkabanJobException.getId()>
+                      <ul class = "list-group">
+                      @for(lines <- azkabanJobException.getLoggingEvent().getLog()) {
+                        <li class="list-group-item"> @for(line <- lines) {&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;@line <br>} </li>
```

Comment:
Rather than specifying multiple non breaking space, you can try to use css to specific something like left padding or left margin.